.eslintrc.js

```diff
@@ -54,14 +54,15 @@ module.exports = {
 
     'import/no-useless-path-segments': 'error',
     'import/order': [
-      'error',
+      'warn',
       {
         groups: [['builtin', 'external'], 'internal', 'sibling', 'parent'],
         'newlines-between': 'ignore',
       },
     ],
     'import/prefer-default-export': 0,
-    'import/extensions': ['error', 'never', { jsx: 'always', json: 'always' }],
+    'import/extensions': ['warn', 'never', { jsx: 'always', json: 'always' }],
+    'no-sequences': ['warn'],
```

Comment:
same here as well, i dont think it would be nice to see a long warn logs in CI 🙏

Reply:
we have sequences in our code base but because of the es lint problem it wasn't throw any error! 
airbnb es lint config that we have in our code base assume it as error but we are using comma expression, so i can't turn it as error, since it's an anti pattern in airbnb es lint config I just can make it as warn or turn it off which is not safe IMO.